lib/discoverable.coffee

```diff
@@ -289,13 +283,6 @@ exports.findServices = Promise.method (services, timeout, callback) ->
 			instance.destroy()
 	.asCallback(callback)
 
-getServiceBrowser = (timeout) ->
```

Comment:
This is the only place where any of the `browserBackends` was used and it is not exported. The only reference to it was removed in the previous PR: https://github.com/balena-io-modules/resin-discoverable-services/pull/40/files#diff-fd11f2fe47e9314a18ca9be3ef6e885b98ba8095fe4831b96f2e8f7045536bc3L242